main.py

```diff
@@ -3,7 +3,7 @@
 import os
 
 asset_items = AssetItems(config.assets_info_file)
-assets: list[Asset] = []
+assets = []
```

Comment:
Because It didn't work :smile: 

`~/ngrave/assets$ python3 main.py 
Traceback (most recent call last):
  File "main.py", line 6, in <module>
    assets: list[Asset] = []
TypeError: 'type' object is not subscriptable

`~/ngrave/assets$ python3 -V
Python 3.8.10
sinova@ubuntu-dsk:~/ngrave/assets$ 
`

Reply:
ah, probably related to older python version, I will change `list` to `typing.List`